src/test/java/uk/gov/hmcts/reform/iahearingsapi/domain/handlers/presubmit/HearingsDynamicListPreparerTest.java

```diff
@@ -90,51 +109,57 @@ void should_write_update_hearings_list() {
 
         assertNotNull(callbackResponse);
         verify(hearingService, times(1)).getHearings(caseId);
-        assertEquals(2, asylumCase
-            .read(CHANGE_HEARINGS, DynamicList.class).get().getListItems().size());
-        assertEquals("1", asylumCase
-            .read(CHANGE_HEARINGS, DynamicList.class).get().getListItems().get(0).getCode());
-        assertEquals("Substantive - 20 January 2023",
-            asylumCase
-                .read(CHANGE_HEARINGS, DynamicList.class).get().getListItems().get(0).getLabel());
-        assertEquals("2", asylumCase
-            .read(CHANGE_HEARINGS, DynamicList.class).get().getListItems().get(1).getCode());
-        assertEquals("Substantive - 21 January 2023",
-            asylumCase
-                .read(CHANGE_HEARINGS, DynamicList.class).get().getListItems().get(1).getLabel());
-        assertEquals(callbackResponse.getData(), asylumCase);
+        assertEquals(asylumCase
```

Comment:
ordering of the params are incorrect here, should be 
public static void assertEquals(Object expected, Object actual) {